spatialmath/testutils.go

```diff
@@ -0,0 +1,62 @@
+package spatialmath
+
+import (
+	"github.com/golang/geo/r3"
+)
+
+// MakeTestBox creates a test box with the given orientation, position, dimensions, and label.
+func MakeTestBox(o Orientation, pt, dims r3.Vector, label string) Geometry {
```

Comment:
Lets revert all this I don't want to add these as public functions.  Its easy enough to add the line `test.That(t, err, test.ShouldBeNil)` where appropriate for packages that do not have access to them